README.md

```diff
@@ -80,6 +80,8 @@ If you have any problems, submit an issue or send a message via gitter.
 
 ### CI
 
+We're using GitHub actions to run our CI process. View a visualisation of the workflow [here](https://lucid.app/lucidchart/invitations/accept/inv_74e30928-4106-49da-b75c-7a6291b225f4).
```

Comment:
Can you change this to the following?

We're using GitHub Actions to run our CI process. View [a visualisation of the workflow here](https://lucid.app/lucidchart/invitations/accept/inv_74e30928-4106-49da-b75c-7a6291b225f4).

Reply:
Yep - done! 

👋 btw!! hope all is well ❤️